### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by kittsville**
- Add _Usage_ section with installation instructions
- Minor typos

Given this repository is public we might want to slap a license on it to encourage wider adoption.

### CODE REVIEW

File: README.md
Comment (rcdmrl) [inline README.md:63]: @l-catallo or @filosganga: Since we're here, are these points correct? 
> In case (3) the library does not allow the signal to be processed again

Case 3 is when the "lock expired" and I thought that the whole reason of this expiration was to allow reprocessing the same signal id when something goes wrong.

Also, for the case 2 we say:
> In cases (1) and (2) the library allows the signal to be processed.
If the `completedAt` is present and we try to process the same signal (same `id`) isn't that a dupicate?

**Comment by rcdmrl** [comment]
Thanks @kittsville! I'm just pinging Phil and Luca to clarify some of the points that were already in the documentation but, other than that, everything looks great!

**Comment by kittsville** [comment]
Any news?

File: README.md
Comment (l-catallo) [reply README.md:63]: Yeah I think case 2 and 3 are swapped.
In case (2) the process is stopped from being executed.
In case (3) the process is executed.

Comment (rcdmrl) [reply README.md:63]: @kittsville Would it be asking too much if we asked you to make this little update? 🙏 I'm also personally happy to approve and update it myself after it gets merged.

Comment (kittsville) [reply README.md:63]: Yeah sure, I'm happy to 👍

**Review by rcdmrl — APPROVED** [review]

**Pull request merged**
